sql code/ksm-reunion-2024.sql

```diff
@@ -615,10 +627,258 @@ and (lower(a.xcomment) LIKE ('%ksm%1st%reunion%')
 vs as (select sol.id_number,
 sol.solicitor_name
 from sol),
+ 
+ 
+--- Anon Donor Honor Role 
+
+/* Parameters -- update this each year
+  Also look for <UPDATE THIS> comment */
+params_cfy As (
+  Select
```

Comment:
-- You can remove the manual update by making this dependent on manual_dates which you're already using
Select cfy As params_cfy
From manual_dates